packages/ag-grid-styles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@equinor/fusion-react-ag-grid-styles",
-  "version": "30.0.0",
+  "version": "30.1.0",
```

Comment:
I thought changeset will update this for you?

Reply:
yes it will, but i like this to have same version as the ag-grid package.

Reply:
which is at 3.2.0 and i set this as a minor release